Source/DafnyLanguageServer/Language/DafnyProgramVerifier.cs

```diff
@@ -52,10 +52,17 @@ DafnyOptions options
 
       cancellationToken.ThrowIfCancellationRequested();
 
+      if (engine.Options.PrintFile != null) {
+        var moduleCount = Translator.VerifiableModules(program).Count();
+        foreach (var t in translated) {
```

Comment:
You can immediately deconstruct the tuple here, so you can better names than `t` and `Item1` and `Item2`